src/app/pages/OpenWalletPage/Features/FromLedger/index.tsx

```diff
@@ -150,6 +151,7 @@ export function FromLedgerModal(props: FromLedgerModalProps) {
             }}
             background={{
               color: 'status-error',
+              dark: theme === 'dark',
```

Comment:
Strange, I thought this would be inherited

Reply:
Seems like it tries to detect color darkness, but doesn't take into account the opacity `color: 'status-error', opacity: 0.3,`. I think we should
```diff
-      'status-error': '#ff4212',
+      'status-error': '#ff42124c', // 30% opacity

-  color: 'status-error', opacity: 0.3, [dark: theme === 'dark',]
+  color: 'status-error',
```

Reply:
(and same with other code involving `opacity:`)

Reply:
I think `status-error` works just fine, but inner components (here hyperlinks) are inheriting wrong theme which defaults to dark.

Reply:
Updated after a meeting
- use 'weak' workaround 
- unify opacity for all boxes

Change set is big, because I added ThemeProvider to one of a snapshot. Currently snapshots have default Grommet theme css rules not ours. I can remove that test update, but I am not sure it's worth having snapshots with css rules that we are not using. 


```
  // old snapshot
  background-color: rgba(255,170,21,0.2); // that's default Grommet  statusColors for example
  border-left: solid 3px #FFAA15;

  // snapshot with ThemeProvider
  background-color: rgba(244,206,75,0.25);
  border-left: solid 3px #f4ce4b;
```

Reply:
Nice

I don't know the main reason for snapshots with css, but it does help detect visual changes from updating dependencies :shrug: